openmetadata-ui/src/main/resources/ui/src/pages/TagPage/TagPage.tsx

```diff
@@ -0,0 +1,305 @@
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import { Button, Col, Row, Tabs } from 'antd';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
+import { useTranslation } from 'react-i18next';
+import { useHistory, useLocation, useParams } from 'react-router-dom';
+import DescriptionV1 from '../../components/common/EntityDescription/DescriptionV1';
+import Loader from '../../components/common/Loader/Loader';
+import { TitleBreadcrumbProps } from '../../components/common/TitleBreadcrumb/TitleBreadcrumb.interface';
+import { AssetSelectionModal } from '../../components/DataAssets/AssetsSelectionModal/AssetSelectionModal';
+import { EntityHeader } from '../../components/Entity/EntityHeader/EntityHeader.component';
+import { EntityDetailsObjectInterface } from '../../components/Explore/ExplorePage.interface';
+import AssetsTabs, {
+  AssetsTabRef,
+} from '../../components/Glossary/GlossaryTerms/tabs/AssetsTabs.component';
+import PageLayoutV1 from '../../components/PageLayoutV1/PageLayoutV1';
+import { FQN_SEPARATOR_CHAR } from '../../constants/char.constants';
+import { EntityType } from '../../enums/entity.enum';
+import { SearchIndex } from '../../enums/search.enum';
+import { Tag } from '../../generated/entity/classification/tag';
+import { MOCK_GLOSSARY_NO_PERMISSIONS } from '../../mocks/Glossary.mock';
+import { searchData } from '../../rest/miscAPI';
+import { getTagByFqn } from '../../rest/tagAPI';
+import { getCountBadge } from '../../utils/CommonUtils';
+import { getEntityName } from '../../utils/EntityUtils';
+import Fqn from '../../utils/Fqn';
+import { getQueryFilterToExcludeTerm } from '../../utils/GlossaryUtils';
+import {
+  getClassificationDetailsPath,
+  getClassificationTagPath,
+} from '../../utils/RouterUtils';
+import {
+  escapeESReservedCharacters,
+  getEncodedFqn,
+} from '../../utils/StringsUtils';
+import './tagpage.less';
+
+const TagPage = () => {
+  const { t } = useTranslation();
+  const { pathname } = useLocation();
+  const history = useHistory();
+  const { tab } = useParams<{ tab: string }>();
+  const tagFqn = pathname.split('/').find((path) => path.includes('.'));
+  const [isLoading, setIsLoading] = useState(false);
+  const [tagItem, setTagItem] = useState<Tag>();
+  const [assetModalVisible, setAssetModalVisible] = useState(false);
+  const [assetCount, setAssetCount] = useState<number>(0);
+  const assetTabRef = useRef<AssetsTabRef>(null);
+  const [previewAsset, setPreviewAsset] =
+    useState<EntityDetailsObjectInterface>();
+  const [breadcrumb, setBreadcrumb] = useState<
+    TitleBreadcrumbProps['titleLinks']
+  >([]);
+
+  const activeTab = useMemo(() => {
+    return tab ?? 'overview';
+  }, [tab]);
```

Comment:
We can assign default tab in line number 58 only

Reply:
Updated.